content/news/posts/2025/breaking-change-guzzle-removal-v9.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  "Breaking Change in V9 Guzzle removal, what are the alternatives?"
+subtitle: "You are using Guzzle in your module. Starting from PrestaShop V9 the library is not available by default any more: how can you adapt your module?"
+date:   2025-03-17
+authors: [ JonathanLelievre ]
+icon: icon-code
+tags: [autoupgrade, module, beta, releases, community, contribution, development, qa]
+---
+
+The PrestaShop V9 version is a major version, as such it comes with breaking changes among which some removed dependencies, you can find more details about those changes on the [developer documentation](https://devdocs.prestashop-project.org/9/modules/core-updates/9.0/), but in this article we're going to focus on one in particular: the removal of Guzzle and how to adapt your module using it.
```

Comment:
```suggestion
PrestaShop 9 is a major release, meaning it [introduces breaking changes](https://devdocs.prestashop-project.org/9/modules/core-updates/9.0/), including the removal of certain dependencies. One of the most impactful changes for module developers is the removal of Guzzle, a popular HTTP client that many modules relied on.

If your module depends on Guzzle, it will no longer be available by default in PrestaShop 9. In this article, we’ll explain why Guzzle was removed, what this means for your module, and how to migrate to a suitable alternative.
```